client/src/endpoint/account.rs

```diff
@@ -43,3 +44,171 @@ mod tests {
         assert_eq!(request.uri().path(), "/accounts/abc123");
     }
 }
+/// An endpoint that accesses the transactions for a specific account
+#[derive(Debug)]
+pub struct Transactions {
+    id: String,
+    cursor: Option<String>,
+    order: Option<Order>,
+    limit: Option<u32>,
+}
+
+impl Transactions {
+    /// Returns a new end point for account transactions. Hand this to the client in order to request
+    /// transactions for a specific account.
+    pub fn new(id: &str) -> Self {
+        Self {
+            id: id.to_string(),
+            cursor: None,
+            order: None,
+            limit: None,
+        }
+    }
+
+    /// Starts the page of results at a given cursor
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// use stellar_client::sync::Client;
+    /// use stellar_client::endpoint::account;
+    /// # use stellar_client::endpoint::transaction;
+    /// # use stellar_client::endpoint::Order;
+    ///
+    /// let client   = Client::horizon_test().unwrap();
+    /// #
+    /// # // grab first page and extract cursor
+    /// # let transaction_ep   = transaction::All::default().limit(1);
+    /// # let txns             = client.request(transaction_ep).unwrap();
+    /// # let txn              = &txns.records()[0];
+    /// # let endpoint         = account::Transactions::new(txn.source_account());
+    /// # let first_page       = client.request(endpoint).unwrap();
+    /// # let cursor           = first_page.next_cursor();
+    /// # let account_id       = txn.source_account();
```

Comment:
a little nit-picky, but can move this up a few rows to use when defining the endpoint.